packages/data-stores/tsconfig.json

```diff
@@ -7,5 +7,9 @@
 	},
 	"include": [ "src" ],
 	"exclude": [ "**/test/*" ],
-	"references": [ { "path": "../i18n-utils" }, { "path": "../domain-utils" } ]
+	"references": [
+		{ "path": "../i18n-utils" },
+		{ "path": "../domain-utils" },
+		{ "path": "../format-currency" }
+	]
```

Comment:
Added `format-currency` to references here to fix [this error](https://github.com/Automattic/wp-calypso/runs/6061382465?check_suite_focus=true):

```log
Error: ➤ YN0000: │ wp-calypso@workspace:. STDOUT packages/data-stores/src/domain-suggestions/utils.ts(1,28): error TS2307:
Cannot find module '@automattic/format-currency' or its corresponding type declarations.
```